cli/cli.go

```diff
@@ -284,6 +290,31 @@ func fetchIndex() {
 	}
 }
 
+func installFromVersionFile() {
+	blob, err := ioutil.ReadFile(VERSION_FILE)
+	if err != nil {
+		fmtc.Println("Cannot read %s", VERSION_FILE)
+		exit(1)
+	}
+
+	version, err := readFirstWord(string(blob))
+	if err != nil {
+		fmtc.Println("Cannot find version in %s", VERSION_FILE)
```

Comment:
Why do you print custom error message if you have error struct with info about error?